js/test/wallet/parseURL.test.ts

```diff
@@ -6,29 +6,32 @@ describe('parseURL', () => {
     describe('parsing', () => {
         describe('when given correct params', () => {
             it('should parse successfully', () => {
-                const url =
-                    'solana:mvines9iiHiQTysrwkJjGf2gb9Ex9jXJX8ns3qwf2kN?amount=0.01&reference=82ZJ7nbGpixjeDCmEhUcmwXYfvurzAgGdtSMuHnUgyny&label=Michael&message=Thanks%20for%20all%20the%20fish&memo=OrderId5678';
+                const recipientPublicKey = 'mvines9iiHiQTysrwkJjGf2gb9Ex9jXJX8ns3qwf2kN';
+                const referencePublicKey = '82ZJ7nbGpixjeDCmEhUcmwXYfvurzAgGdtSMuHnUgyny';
+                const url = `solana:${recipientPublicKey}?amount=0.01&reference=${referencePublicKey}&label=Michael&message=Thanks%20for%20all%20the%20fish&memo=OrderId5678`;
 
                 const parsed = parseURL(url);
 
-                expect(parsed.recipient).toEqual(new PublicKey('mvines9iiHiQTysrwkJjGf2gb9Ex9jXJX8ns3qwf2kN'));
+                expect(new PublicKey(recipientPublicKey).equals(parsed.recipient)).toBe(true);
                 expect(parsed.amount).toEqual(new BigNumber(0.01));
                 expect(parsed.token).toBeUndefined();
-                expect(parsed.references).toEqual([new PublicKey('82ZJ7nbGpixjeDCmEhUcmwXYfvurzAgGdtSMuHnUgyny')]);
+                expect((parsed.references as PublicKey[]).length).toBe(1);
+                expect(new PublicKey(referencePublicKey).equals((parsed.references as PublicKey[])[0])).toBe(true);
```

Comment:
Casting like this is weird -- `expect(parsed.references?.length).toBe(1)` and `parsed.references?.[0]` does the same thing right? Or just use the `!` operator if you know it won't be `undefined`.